src/main/java/frc/robot/RobotPreferences.java

```diff
@@ -325,14 +325,14 @@ public static final class ThreeCargo {
       public static final SN_DoublePreference shooterRPM1_6 = new SN_DoublePreference(
           "shooterRPM1_6", ShooterPrefs.shooterPresetUpperFenderRPM.getValue());
       public static final SN_DoublePreference hoodAngle1_6 = new SN_DoublePreference(
-          "hoodAngle1_6", 5);
+          "hoodAngle1_6", HoodPrefs.hoodOldLow.getValue());
```

Comment:
FYI, if you change hoodOldLow via RobotPreferences you will need to restart the code for it to be updated for hoodAngle1_6

Reply:
Hm, that's not ideal. It would be nice if there were a better way for the coded value of an SN_Preference to be the value of another SN_Preference. It's okay for now though.